_episodes/20.md

```diff
@@ -0,0 +1,370 @@
+   in Trino. The build excludes some modules, runs the compiler on multiple 
+   threads, skips the tests, javadocs, and the Airlift code style checks. If you
+   would like to run code style check on a specific module (e.g. 
+   trino-elasticsearch) then you can run the following command.
+   
+   ```
+   ./mvnw -pl ':trino-elasticsearch' clean install \
+   -TC2 -nsu \
+   -DskipTests \
+   -Dmaven.javadoc.skip=true \
+   -Dmaven.source.skip=true 
+   ```
+
+1. Sign the CLA.
+    
+    Sign the [contributor license agreement (CLA)](https://github.com/trinodb/cla/blob/master/Trino%20Foundation%20Individual%20CLA.pdf) 
+    to agree that all of your code you commit to the project is subject to the 
+    Apache License 2.0. Once you sign the agreement, scan and submit the form to
+    [cla@trino.io](mailto:cla@trino.io). This email gets checked every few days,
+    and you can check if your name has been added to the [contributors](https://github.com/trinodb/cla/blob/master/contributors)
+    list.
+    
+1. At this point you will want to look for an [issue labeled "good first issue"](https://github.com/trinodb/trino/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22)
+This identifies issues that we think are more approachable for developers that 
+aren't as familiar with the Trino repository yet.
+
+1. One final thing before you move on to the contribution process. Before you
+start jumping in and changing the code, you'll also want to create a special
+branch for your changes. A branch in git makes a separate workflow for all the 
+changes you make to be isolated so that if something goes wrong, or you need to 
+compare with an older branch you can do so. The default branch may either be
+named `master` or `main`. Due to the connotation of the word `master` with 
+slavery, `main` is preferred but you will so both branch names in the wild. See
+[more on branching in git](https://git-scm.com/book/en/v2/Git-Branching-Basic-Branching-and-Merging).
+
+To make a branch for your feature, you can run the following command:
+
+```
+git checkout -b my-feature-branch
+```
+
+1. Follow the remaining steps in the [contribution process page](https://trino.io/development/process.html).
+
+## Question of the week: How do I remove nulls from an array in Trino?
+
+A [question posted to StackOverflow](https://stackoverflow.com/questions/66162776) 
+asked the following question:
+
+> I'm extracting data from a json column in Trino and getting the output in an 
+array like this `[AL,null,NEW]`. The problem is I need to remove the null since
+the array has to be mapped another array.I tried several options but no luck.
+How can I remove the null and get only `[AL,NEW]` without unnesting?
+
+[Piotr Findeisen](https://twitter.com/findepi) replied: 
+
+You can use [filter()](https://trino.io/docs/current/functions/array.html#filter)
+for this:
+
+```
+trino> SELECT filter(ARRAY['AL',null,'NEW'], e -> e IS NOT NULL);
```

Comment:
`SELECT filter(ARRAY['AL',null,'NEW'], e -> e IS NOT NULL);`
-> 
`SELECT filter(ARRAY['AL', NULL, 'NEW'], e -> e IS NOT NULL);`

minor: the space between the array items could be added and the switch from `null` to `NULL`  would be consistent with the rest of the trino docs.